index.src.html

```diff
@@ -1885,30 +1889,41 @@
 ---------------------------------------------
 <xmp class='idl'>
 enum HardwareAcceleration {
-  "allow",
-  "deny",
-  "require",
+  "no-preference",
```

Comment:
As discussed in https://github.com/w3c/webcodecs/issues/239, I tend to prefer a more descriptive default value.
@dalecurtis mentionned renaming no-preference to prefer-efficiency.

Reply:
I defer final decision to @aboba @padenot, I prefer `no-preference`.

Reply:
no-preference is fine. I've also seen "balanced" used.

Reply:
I find this rather descriptive. In passing this value, the author explicitly signals that they don't have any particular preference, and defers to the UA to make the decision on what decoder to use, based on the scenario characteristics (on battery power, video resolution, etc.).